models/src/main/java/com/vimeo/networking2/Document.kt

```diff
@@ -0,0 +1,13 @@
+package com.vimeo.networking2
+
+/**
+ * Document data.
+ */
+data class Document(
+
+    /**
+     * The partially stripped html for the terms of service.
```

Comment:
Docs reference TOS but the class name is generic. Is this class only used for TOS or is it used for other docs as well. If other docs, I would suggest changing the documentation wording to `...for documents like the terms of service`